e2e/E2E-TESTS.md

```diff
@@ -0,0 +1,38 @@
+# E2E Test Inventory
+
+#### Fields Description
+
+- Command: the MongoDB CLI command without `mongocli atlas|om|cm` 
```

Comment:
@gssbzn @colm-quinn Could you tell me what you think about this format? I would like to gather some feedback before adding all the commands

Reply:
you can have a look at the file here https://github.com/mongodb/mongocli/blob/e2e_test_audit/e2e/E2E-TESTS.md

Reply:
I like the inventory and markdown table. I'd probably summarise the Y/N steps into a single line so it reads more easily. Something like:

```
- E2E Atlas: 
    -  Possible values: ['Y'|'N']
    - Indicates if an e2e test for the command is present ('Y') or not present ('N') for Atlas
...
```